projects/igniteui-angular/src/lib/grids/grid/grid.pinning.spec.ts

```diff
@@ -417,6 +418,41 @@ describe('IgxGrid - Column Pinning #grid', () => {
             expect(grid.pinnedColumns.length).toEqual(1);
             expect(grid.unpinnedColumns.length).toEqual(2);
         }));
+
+        it('should allow navigating to/from pinned area', (async() => {
+            pending('https://github.com/IgniteUI/igniteui-angular/pull/6910');
+            const fix = TestBed.createComponent(DefaultGridComponent);
+            fix.detectChanges();
+            const grid = fix.componentInstance.instance;
+
+            const cellContactName = grid.getCellByColumn(0, 'ContactName');
+            const range = {
+                rowStart: cellContactName.rowIndex,
+                rowEnd: cellContactName.rowIndex,
+                columnStart: cellContactName.visibleColumnIndex,
+                columnEnd: cellContactName.visibleColumnIndex
+            };
+            grid.selectRange(range);
+            grid.navigation.activeNode = {row: cellContactName.rowIndex, column: cellContactName.visibleColumnIndex};
+            fix.detectChanges();
+
+            const keydownArrowRightEvent = new KeyboardEvent('keydown', {key: 'ArrowRight'});
+            grid.navigation.dispatchEvent(keydownArrowRightEvent);
```

Comment:
You can use:
grid.navigation.dispatchEvent(UIInteractions.getKeyboardEvent('keydown', 'ArrowDown'))